dbms/programs/client/Client.cpp

```diff
@@ -171,6 +242,9 @@ class Client : public Poco::Util::Application
     /// External tables info.
     std::list<ExternalTable> external_tables;
 
+    /// Suggestion limit for how many databases and tables to fetch
+    int suggestion_limit = 100;
```

Comment:
why so low by default? For databases that can be reasonable, for tables - a bit too low, for columns definitely too low.

Reply:
@filimonov what limit will you suggest? twice the amount? not really sure :(

Reply:
@filimonov Bumped limit to 256. One can always adjust it to higher/lower limits by adding --suggestion_limit=<value> to your CLI client on startup ex. ./clickhouse client --suggestion_limit=512